README.md

```diff
@@ -1,16 +1,60 @@
+
+- Central DB as a service that cannot be fully configured for a particular use; data mesh is a pattern that is supplanting the one DB to rule them all of the past
+
+- Monolithic architectures that require the software to be released at best quarterly as opposed to minute by minute in a high performing organization
+
+Here are a few practical implications for the design of our software and services that we may not have fully embraced in the past:
+
+- *Automation is fundamental*: human interaction to tweak the settings means treating the containers as pets, not cattle.  That is an anti-pattern not consistent w/ cloud-native and Container Attached Storage.
+
+- *Design for failure*:  Kubernetes and the cloud-native pattern assumes that everything will fail at some point. Chaos engineering such as the Litmus Chaos project we started and developed into a viable project enforces the need to design for failure by creating failures in systems.
+
+- *GitOps*: Building off the importance of automation - the desired state of the environment is in the repository, i.e. GitOps is fundamental.
+
+- *Loosely Coupled* may be the two most important words in [the CNCF’s definition of cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md). The need for loose coupling is one reason why our users like Steven Bower do not want to use shared storage. One video we have in the welcome and onboarding slides is [this interview with Steven](https://www.youtube.com/watch?v=z_LbRfDKPvE).
+
+- *APIs and Declarative approach*: Loosely coupled and the existence of many sub-components suggests there are clear APIs including within the product and typically declarative approaches are used
+
+- *Cloud operations including auto-scaling*: Cloud-native also tends to imply cloud scalability - the ability to scale UP and scale DOWN.
+
+We will know we are succeeding in enabling our personas when a standard user of Kubernetes can grab OpenEBS and:
+
+- *grok it* -- what it is supposed to do, how it is designed, how it is explained
+
+- and *operate it* -- knowing that it will behave in a cloud-native or container attached storage way
+
+…while the Kubernetes SREs within the Platform Team can support the use of OpenEBS at scale because they will be able to do their job. We have more work to do here in defining what these platform teams require -- thankfully we are in frequent contact with them at places such as Bloomberg, Flipkart and Optoro. 
+
+Last but certainly not least -- we are working with data and increasingly important data that needs to be secured at least for some period of time.  We must avoid losing data, understanding that solutions such as flavors of OpenEBS LocalPV themselves by design do nothing to secure the data.  Remember the old saying, when our users lose their data they typically lose their jobs.
```

Comment:
The above needs to be rephrased or made clear that there are workloads that only need Local PV and the motivation from MayaData to work on Local PV. Inserting the data loss alongside Local PV raises the question if we should really work on Local PV at all. The other aspects of Cloud Native, as described in the many OpenEBS adoption stories indicate that teams are depending on cloud native applications to have data protection capabilities. 

Ref: 
* https://github.com/openebs/openebs/blob/master/adopters/optoro/README.md
* Cloud Native DR - https://docs.google.com/presentation/d/1GlVI8OjxNcEp3hqe-rjItTfLiuvi_RD4cTX33iyOe0A/edit#slide=id.p

Reply:
@kmova can you please propose the wording? Alternatively, we can merge this PR and take it as followup ticket/PR. What would you prefer?

Reply:
I think it is clear as written that by design OpenEBS LocalPV is not designed to have replicas and secure the data.